packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -346,7 +356,7 @@ export interface ITelemetryContext {
 	 * @param property - property name of the telemetry data being tracked (ex: "DirectoryCount")
 	 * @returns undefined if item not found
 	 */
-	get(prefix: string, property: string): TelemetryEventPropertyType;
+	get(prefix: string, property: string): TelemetryEventPropertyTypeExt;
```

Comment:
I wish we could avoid exposing the `Ext` types on any public API.  Ideally `ITelemetryContext` wouldn't be public but it currently has to be.

The reason that's a bummer is that then we can't easily expand the supported types (like we discussed allowing arbitrary Jsonable).  _If_ it's only used as an "in" type (users of public API pass _in_ the telemetry event) then we actually can expand what it accepts and it'll be automatically backwards compatible (can do in a minor bump).  So I'd like to keep `get` here as returning the non-`Ext` one for now.  I am gonna put out a small PR to deprecate it, actually. This should be a "write-only" API, like loggers (they only have `send`).

Reply:
PR to deprecate get/serialize on here:  #19409

For now, can you avoid changing the return type of `get`? You can do something quick to convert like `return typeof value === "object" ? JSON.stringify(value) : value;` (as opposed to the big switch statement and throwing in the default case etc - it just won't happen)